plugins/shared/cmd/launcher/command/device.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/hashicorp/nomad/plugins/device"
 	"github.com/kr/pretty"
 	"github.com/mitchellh/cli"
+	"github.com/vmihailenco/msgpack"
```

Comment:
```suggestion
	"github.com/zclconf/go-cty/cty/msgpack"
```

Reply:
wrong msgpack import results in: 
> 2019-11-26T01:53:09.050Z [ERROR] nomad-plugin-launcher: failed to set config: error="rpc error: code = Unknown desc = SetConfig failed: some_optional_string_with_default was not acceptible, cannot be empty%!(EXTRA string=value, string=)"

this binary can be built and tested using the instructions at: 
https://github.com/hashicorp/nomad-skeleton-device-plugin